dist/znk.infra.js

```diff
@@ -864,6 +879,179 @@
 })(angular);
 
 
+(function (angular) {
+    'use strict';
+
+    angular.module('znk.infra.storage').factory('storageFirebaseAdapter', [
+        '$firebaseAuth', '$firebaseObject',
```

Comment:
is there a real need for $firebaseAuth ? maybe we should start working with Firebase only